.github/workflows/ci.yml

```diff
@@ -9,54 +9,63 @@ on:
 concurrency:
   group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
   cancel-in-progress: true
-  
+
 jobs:
   test:
     name: +test
-    runs-on: [earthly-satellite#gha-lib] # https://docs.earthly.dev/earthly-cloud/satellites/gha-runners
+    runs-on: ubuntu-latest
```

Comment:
Explicit version `ubuntu-24.04`? To ensure the latest version does not break our builds.

@renovate-bot can bump it up when a new version is out.